cfme/fixtures/authentication.py

```diff
@@ -31,39 +31,6 @@ def auth_provider(prov_key):
     return authutil.get_auth_crud(prov_key)
 
 
-@pytest.fixture(scope='function')
-def auth_user_data(auth_provider, user_type):
```

Comment:
moved to a utils method, the fixture replacing this now returns a single auth_user object created from the yaml data returned by the function in a util context.